discord/guild.py

```diff
@@ -29,39 +29,9 @@ class GuildLimit(NamedTuple):
 
 
 class Guild(DiscordObject):
-    __slots__ = (
-        "region",
-        "owner_id",
-        "mfa_level",
-        "name",
-        "id",
-        "_members",
-        "_channels",
-        "_vanity",
-        "_banner",
-    )
-
-    _roles: set[Role]
-    me: Member
+    owner: bool = False
     owner_id: Snowflake
-
-    def __init__(self, data: GuildPayload):
-        self._members: dict[Snowflake, Member] = {}
-        self._channels: dict[Snowflake, Union[TextChannel, VoiceChannel]] = {}
-        self._roles = set()
-
-    def _add_channel(self, channel: Union[TextChannel, VoiceChannel], /) -> None:
-        self._channels[channel.id] = channel
-
-    def _delete_channel(self, channel: DiscordObject) -> None:
-        self._channels.pop(channel.id, None)
-
-    def add_member(self, member: Member) -> None:
-        self._members[member.id] = member
-
-    def add_roles(self, role: Role) -> None:
-        for p in self._roles.values:
-            p.postion += not p.is_default()
-            # checks if role is @everyone or not
-
-            self._roles[role.id] = role
+    members: List[dict]
```

Comment:
We could do it in a way that we can either access a member using the member["id"] or member[index]
I think it will be more versatile

Reply:
maybe `guild.get_member`?